src/app/subject/services/volume-data.service.ts

```diff
@@ -56,34 +43,33 @@ export class VolumeDataService {
     }
 
     return this.http
-      .post<any>(
-        this.parseURL(studyName, subjectId, descriptiveStatistic),
-        JSON.stringify({ sources: new_sources }),
-        {
-          headers: new HttpHeaders({
-            'Content-Type': 'application/json'
-          })
-        }
-      )
+      .post<any>(this.parseURL(), JSON.stringify({ sources: new_sources }), {
+        headers: new HttpHeaders({
+          'Content-Type': 'application/json'
+        })
+      })
       .pipe(takeUntil(this.destroy$))
   }
 
-  private parseURL(studyName, subjectId, descriptiveStatistic): string {
+  private parseURL(): string {
     let url = [
       this.url,
-      studyName,
-      subjectId,
-      DescriptiveStatistic[descriptiveStatistic].toLowerCase()
+      this.options.studyName,
+      this.options.subjectId,
+      DescriptiveStatistic[11].toLowerCase()
```

Comment:
Can this `11` be documented or passed as a defined constant?

Reply:
Yes I forgot to replace this. This was originally the variable name. I replaced it for testing purposes. I'll fix it. Thanks.